scripts/hail_batch/eqtl_hail_batch/generate_eqtl_spearman.py

```diff
@@ -162,17 +140,6 @@ def spearman_correlation(df):
     return spearman_df
 
 
-backend = hb.ServiceBackend(billing_project='tob-wgs', bucket='cpg-tob-wgs-test')
-b = hb.Batch(name='eQTL', backend=backend, default_python_image=DRIVER_IMAGE)
-
-spearman_dfs_from_scatter = []
-for i in range(get_number_of_scatters()):
-    # for i in range(5):
-    j = b.new_python_job(name=f'process_{i}')
-    result: hb.resource.PythonResult = j.call(run_computation_in_scatter, i)
-    spearman_dfs_from_scatter.append(result)
-
-
 def function_that_merges_dataframes(*df_list):
```

Comment:
Might want to rename this function `merge_df_and_convert_to_string` just to make the meaning of it super clear.

Reply:
good idea 👌